metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/ELASTICSEARCH/2.3.3/configuration/elastic-site.xml

```diff
@@ -47,7 +47,15 @@
         <name>path_data</name>
         <value>"/opt/lmm/es_data"</value>
         <description>Path to directory where to store index data allocated for this node. e.g. "/mnt/first", "/mnt/second"</description>
-    </property>    
+    </property>
+    <property>
+        <name>http_cors_enabled</name>
+        <value>"false"</value>
```

Comment:
We have to have the quotes, otherwise Ambari will change treat it as a boolean which will be written to False in elasticsearch.yml and Elasticsearch doesn't recognize that as False for this parameter only.
Please don't make them all the same in a clean-up pr, it will break things that work.